docker/Dockerfile.processor

```diff
@@ -12,6 +12,7 @@ COPY . /app/
 USER app
 
 RUN mkdir -p /tmp/symbols/cache
+RUN mkdir -p /tmp/symbols/tmp
```

Comment:
That makes a lot of sense. I hadn't looked at the stackwalker code, yet.

We could fix it, but I don't think it's necessary. One of the things that made this hard is that the `BreakpadStackwalkerRule2015` doesn't say anything about `stackwalker` execution. It's completely silent in all the error situations I've seen so far which is unhelpful. If you see something, log something. I'll probably fix that at some point.

Thank you!